tests/apps/ruby3.4/sam-test-app/sam-test-app/template.yaml

```diff
@@ -0,0 +1,39 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  sam-test-app11
```

Comment:
Should we indicate in the description that this template is for Ruby 3.4?

Reply:
This is just for our own tests, so I don't think it's important really. It's visible from the complete path (`tests/app/ruby3.4`).